pkg/machinepool/machinepool.go

```diff
@@ -1061,12 +1061,39 @@ func (m *machinePool) ListMachinePools(r *rosa.Runtime, clusterKey string, clust
 	// Create the writer that will be used to print the tabulated results:
 	writer := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
 
-	finalStringToOutput := getMachinePoolsString(r, machinePools, args)
+	// Get headers and data
+	headers, tableData := getMachinePoolsData(r, machinePools, args)
 	if isHypershift {
-		finalStringToOutput = getNodePoolsString(nodePools)
+		headers, tableData = getNodePoolsData(nodePools)
+	}
```

Comment:
In the original functions (`getMachinePoolsString` and `getNodePoolsString`), we return a string such as

`test1\ttest2\ttest3\nval1\tval2\tval3\n`

Couldn't we convert this into a list and not change the original functions?